src/ui/SimpleFilter/SimpleFilter.ts

```diff
@@ -458,6 +470,21 @@ export class SimpleFilter extends Component {
     return this.circleElement.el;
   }
 
+  public buildClearElement(): HTMLElement {
+    this.clearElement = $$('button', { className: 'coveo-simplefilter-eraser' }, SVGIcons.icons.mainClear);
+    this.clearElement.hide();
+
+    this.clearElement.setAttribute('aria-label', l('Clear', this.options.title));
```

Comment:
These attributes could be set when you create the button.
eg: 

```
 this.clearElement = $$('button', { 
   className: 'coveo-simplefilter-eraser',
   title: l('Clear', this.options.title),
   'aria-label': l('Clear', this.options.title)
}, SVGIcons.icons.mainClear);
```

Reply:
Cool